src/routes/(infoPages)/[topTierPage]/+page.server.ts

```diff
@@ -150,12 +204,33 @@ export const load = async ({
         )
       : undefined;
 
-    const [heroImageBlurhash, featuredServices, youtubeVideoData] = await Promise.all([
+    // Get related news and events based on the "subject" tag.
+    // There is a tag that corresponds to each top tier, based on the slug.
+    const tag = `subject-${slug}`;
+    const newsOldestDate = getStartOfDayForDateInTZ(
+      getDateSixMonthsAgoInTZ(eventIANATimezone),
+      eventIANATimezone,
+    );
+    const eventStartDate = getStartOfDayForDateInTZ(
+      getCurrentDateInTZ(eventIANATimezone),
+      eventIANATimezone,
+    );
+    const taggedDataPromise = await client.fetch<TaggedNewsAndEventsQuery>(
+      printQuery(taggedNewsAndEventsQuery),
+      {
+        variables: { tag, newsOldestDate, eventStartDate },
+      },
+    );
+
+    const [heroImageBlurhash, featuredServices, youtubeVideoData, taggedData] = await Promise.all([
       heroImageBlurhashPromise,
       Promise.all(featuredServicesPromises).then((arr) => arr.flat()),
       youtubeVideoDataPromise,
+      taggedDataPromise,
     ]);
 
+    if (!taggedData) break fetchData;
```

Comment:
Should we fail `fetchData` if the `taggedData` doesn't come thru, or should we gracefully degrade the page?

Reply:
Good question; my thinking was that if `taggedData` doesn't come through it means the request failed (not that there was no tagged data, since that should be represented by zero-element arrays for `newsCollection.items` and `eventEntryCollection.items`), which probably means there's a larger problem.  I suppose it makes sense to try to render what we can if only this request fails.  Will update!

Reply:
Updated in bea86b2 so that we still render the page even if this request fails. Thanks for the call-out on this. Going to go ahead and merge now.